schema.sql

```diff
@@ -1,7 +1,12 @@
 /* Database schema analogical to the structure of the classes in our app. */
 
 -- Create book table 
```

Comment:
Kindly add `books` table and create 1-to-many relationships between your `books` and [`genres`, `authors`, `labels`] tables 
**Hint** : You can use foreign keys in your `books` table to create this 1:M relationships

Reply:
Thanks for your feedback! 
![](https://media1.giphy.com/media/WsRFoTvgBjA7zE5EZi/giphy.gif)

Reply:
This issue has been addressed well. Good job! 👏🏼 👏🏼